examples/interactive.ts

```diff
@@ -15,6 +15,13 @@ const results = await interaction.interact([
     type: InteractType.numeric,
     bold: false,
   },
+  {
+    name: "alpha",
+    message: "enter a key",
```

Comment:
Replace this example with a more realistic one.
like: `type your favorite color`